prometheo/datasets/worldcereal.py

```diff
@@ -516,18 +518,99 @@ def _dekad_startdate_from_date(dt_in):
     return startdate
 
 
-def _predictor_from_xarray(x: xr.DataArray, epsg: int) -> Predictors:
-    # extract the latlons
+def _predictor_from_xarray(arr: xr.DataArray, epsg: int) -> Predictors:
+    def _get_timestamps() -> np.ndarray:
```

Comment:
nit: it might be nice to test this function, since the years, months and days functions seem a bit finnicky